cmd/http.go

```diff
@@ -24,6 +21,8 @@ func handleGetTasksList(w http.ResponseWriter, r *http.Request) {
 		co = r.Context().Value("core").(*core.Core)
 	)
 
+	lo.Debug("received http request", "handler", "get task list")
```

Comment:
hm, these English log lines are superfluous and not helpful. It should be a `chi` middleware logger that actually logs the requests info. Eg: `timestamp  GET   /path?params resp code`, like a standard access log.

Reply:
I have added the debug logs inside the chi middleware to log the common request fields. Let me know if anything has to be added or omitted